### DISCUSSION TITLE
申请退款, 蓝灯太慢,无法使用

### DISCUSSION THREAD

**Discussion by xiangnanscu** [getlantern/lantern-forum-cn#1076] [General]
最后一根稻草是北京时间12月8日晚上, 简直是奇慢无比. 根本没法用. 请退款.
邮箱:280145668@qq.com
log/ticket号码: 80037643

**Comment by MaxMKMax** [comment]
@xiangnanscu 您好. 请您也提供我支付单据. 谢谢!

**Reply by xiangnanscu** [reply]
![1733786452357_d](https://github.com/user-attachments/assets/d6000c9f-a24f-468e-ab37-a91165e20617)

**Reply by xiangnanscu** [reply]
@Mike777Khoo 可以了吗？
